lib/jormungandr/test/data/jormungandr/genesis.yaml

```diff
@@ -3062,3 +3062,11 @@ initial:
   - fund:
     - address: ca1swl53wlqt5dnl63e0gnf8vpazgt6g5mq384dmz72329eh4m8z7e5un8q6lg
       value: 100000000000
+  - fund:
+      - address: ca1s5nwh0ur9qj4tg5hx334ht9zvqwn9ddz8rv5acwdl997zmqpswdujuwqu9j
+        value: 100000000000000
+      - address: ca1s5498yesws3eee80nxlxqlqetpdstam9x0hkcs7f2k4386uyulydvmzrtja
```

Comment:
We need the stake-pool to have stake delegated to it. I added these addresses to the end of the list of funds, such that they won't get picked by the faucet.